src/libOpenImageIO/maketexture.cpp

```diff
@@ -466,6 +466,12 @@ bump_to_bumpslopes(ImageBuf& dst, const ImageBuf& src,
 
     bump_filter = &sobel_gradient<SRCTYPE>;
 
+    float res_x = 1.0f;
+    float res_y = 1.0f;
+
+    const std::string& bumpformat = configspec.get_string_attribute(
```

Comment:
I'm a little suspicious of this line. get_string_attribute returns a string_view, not a `string&`, so I think this is making a temporary std::string and then having bumpformat be a const ref to it? But the lifetime of that string is a little worrying. I would maybe sleep better at night if this line were

    string_view bumpformat = configspec.get_string_attribute ...

Reply:
fixed in 6736d236ae375501eaf5a6bf6fb698c92746d04c